docs/eturnal.md

```diff
@@ -0,0 +1,149 @@
+# eturnal configuration
+
+## Install and setup _eturnal_ as TURN server
+
+Quick-Test instructions in a [Linux Shell](https://github.com/processone/eturnal/blob/master/QUICK-TEST.md) or with [Docker](https://github.com/processone/eturnal/blob/master/docker-k8s/QUICK-TEST.md) are available as well.
+
+#### 1. Download and install
+
+eturnal is available from a variety of sources such as native package managers, binary packages, installation from source or [container image](https://eturnal.net/documentation/code/docker.html). They are all described [here](https://github.com/processone/eturnal#installation).
+
+
+#### 2. Make eturnal run as daemon on startup
+
+- On Linux systems, the eturnal server is usually invoked by systemd. 
+    ```
+    sudo systemctl status eturnal
+    ```
+
+- For non-systemd platforms, example init and OpenRC scripts are shipped below the `etc` directory. 
+  
+  For controlling eturnal, the eturnalctl command can be used; see:
+    ```
+    eturnalctl help
+    ```
+
+##### Running eturnal on privileged ports
+
+On systemd-managed systems, the systemd unit grants `NET_BIND_SERVICE` capability to the [service](https://github.com/processone/eturnal/blob/29e82f260d369a39bd4a395cb981e914b141875b/config/eturnal.service#L23).
+
+Depending on the system configuration Linux kernel capabilities could be used to overcome this limitation. Capabilities can be associated with executable files using _setcap_, so you could allow eturnal's process manager _beam.smp_ in eturnal's `lib` directory, e.g.: 
+```
+setcap 'cap_net_bind_service=+ep' $(find /opt/eturnal -name beam.smp)
+```
+
+#### 3. Configure `eturnal.yml` for usage with Nextcloud Talk
+
+- Next you need to adjust eturnal's configuration file in `/etc/eturnal.yml` to work with Nextcloud Talk. This file uses the (indentation-sensitive!) YAML format. The shipped configuration file contains further explanations.
+- Choose the listening port (default is _3478_) and an authentication secret, where a random hex is recommended
+    ```
+    openssl rand -hex 32
+    ```
+
+- Then uncomment/edit the following settings accordingly:
+
+  ```yaml
```

Comment:
![grafik](https://github.com/nextcloud/spreed/assets/213943/e4de10d3-8003-4f55-9f0b-355f606b29c3)


Doesn't seem to work well.

(you can build the docs locally with `mkdocs build`

Reply:
yeah, that looks a bit ugly. Thanks for the hint with `mkdocs build`. I will check.